test/TestUtils.py

```diff
@@ -185,3 +186,53 @@ def test_expand_paths_vars(monkeypatch):
     monkeypatch.setenv('TEST_PATH', test_path)
     assert utils.expand_paths_vars(['~']) == [os.path.expanduser('~')]
     assert utils.expand_paths_vars(['$TEST_PATH']) == [test_path]
+
+
+@pytest.mark.parametrize(
+    'reset_env_var, message_prefix',
```

Comment:
this will fix the linter violation
```suggestion
    ('reset_env_var', 'message_prefix'),
```